package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@datadog/native-metrics",
+  "version": "1.0.0",
+  "description": "Native metrics collector for libuv and v8",
+  "main": "index.js",
+  "repository": "git@github.com:DataDog/dd-native-metrics-js.git",
+  "author": "Datadog Inc. <info@datadoghq.com>",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/DataDog/dd-native-metrics-js#readme",
+  "scripts": {
+    "install": "(node scripts/should_rebuild && node-gyp-build) || exit 0",
+    "rebuild": "node-gyp rebuild",
+    "prebuild": "node scripts/prebuild.js",
+    "prebuilds": "node scripts/prebuilds.js",
+    "prepublishOnly": "node scripts/prepublish.js",
+    "lint": "node scripts/check_licenses.js && eslint .",
+    "test": "mocha 'test/**/*.spec.js'"
```

Comment:
Have you tried with `--parallel`?

Reply:
I haven't, but there is only a single test file so there would be no benefit even if it worked.